wurst/objediting/AbilityObjEditing.wurst

```diff
@@ -452,13 +453,26 @@ public class AbilityDefinitionPoisonArrows extends AbilityDefinition
 		def.setLevelsDataUnreal("Poa3", lvls, 3, lc)
 		addTooltipProperty("Attack Speed Factor", lc)
 
-	function setStackingType(int level, string value)
-		def.setLvlDataString("Poa5", level, 5, value)
+	function setStackingType(int level, int value)
+		def.setLvlDataInt("Poa5", level, 4, value)
 
-	function presetStackingType(StringLevelClosure lc)
-		def.setLevelsDataString("Poa5", lvls, 5, lc)
+	function presetStackingTypes(IntLevelClosure lc)
+		def.setLevelsDataInt("Poa5", lvls, 4, lc)
 		addTooltipProperty("Stacking Type", lc)
 
+	function presetStackingType(StackingType stackingType, boolean flag)
+		let pow = stackingType castTo int
+		if stackingTypesSet.get(pow)
+			if not flag
+				stackingTypesSet = stackingTypesSet.reset(pow)
+		else if flag
+			stackingTypesSet = stackingTypesSet.set(pow)
```

Comment:
I was just walking around and suddenly looked here. Why not `stackingTypesSet.set(stackingType castTo int, flag)`?

Reply:
You're right. `stackingTypesSet.set(stackingType castTo int, flag)?` would be better.
I copied the legacy codes template for bitset.

Reply:
@rhazarian you wrote it like this initially. Lol.
https://github.com/wurstscript/WurstStdlib2/commit/9559ad20266a0417ab9eda1db5a989f4aac70939

Reply:
I have copied it from Channel options preset 😅 and didn’t think about this better way that time for a reason

Reply:
I have copied it from Channel options preset 😅 and didn’t think about this better way that time for a reason

Reply:
I have copied it from Channel options preset 😅 and didn’t think about this better way that time for a reason

Reply:
I have copied it from Channel options preset 😅 and didn’t think about this better way that time for a reason

Reply:
Roland will you make the change?

Reply:
@rhazarian This is a legacy issue, and was required for the old bitset because it worked by adding/subtracting power of twos from an int. Now that it uses `BlzBitOr`, this short version becomes possible. It will still break this for older patches though.